ContentAwareGenerator.php

```diff
@@ -155,9 +155,9 @@ protected function getBestLocaleRoute(SymfonyRoute $route, $parameters)
      * @param array $parameters which should contain a content field containing
      *                          a RouteReferrersReadInterface object
      *
-     * @return SymfonyRoute the route instance
-     *
      * @throws RouteNotFoundException if no route can be determined
+     *
+     * @return SymfonyRoute the route instance
```

Comment:
Will try and either remove the CS fix or maybe there is a way to specify
the order.

On Mon, Dec 07, 2015 at 02:23:31AM -0800, David Buchmann wrote:
>    In [1]ContentAwareGenerator.php:
> 
>  >       * @throws RouteNotFoundException if no route can be determined
>  > +     *
>  > +     * @return SymfonyRoute the route instance
> 
>    that seems wrong to me as well. i think just about every code i used had
>    the param / return / throws order.
> 
>    —
>    Reply to this email directly or [2]view it on GitHub.
> 
>    Reverse link: [3]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/symfony-cmf/Routing/pull/159#discussion_r46805162
>    2. https://github.com/symfony-cmf/Routing/pull/159/files#r46805162
>    3. https://github.com/symfony-cmf/Routing/pull/159/files#r46805162